lib/cdo/redshift_import.rb

```diff
@@ -40,12 +40,15 @@ def self.complete_table_import(schemas)
   # Returns Array of Redshift table names.
   def self.temporary_import_tables(schema)
     redshift_client = RedshiftClient.instance
+    # Avoid Redshift system table pg_table_def because it contains some rows that label primary keys as tables.
+    # https://docs.aws.amazon.com/redshift/latest/dg/r_PG_TABLE_DEF.html
```

Comment:
I'm not sure I understand this comment- the linked documentation is pretty clear that [`PG_TABLE_DEF`](https://docs.aws.amazon.com/redshift/latest/dg/r_PG_TABLE_DEF.html) stores information about table _columns_, not tables. If the rows in that table are columns, how is a row referencing a primary-key column being incorrectly labelled as a 'table'?

Reply:
Yup. The comment is confusing. I've tried to add some clarification to it, and that also exposed an error in the new query (`DISTINCT` is no longer needed).